mlir/include/mlir/Dialect/Linalg/IR/LinalgRelayoutOps.td

```diff
@@ -57,7 +57,8 @@ class Linalg_RelayoutOp<string mnemonic, list<Trait> traits = []> :
     /// tile factors.
     DenseMap<int64_t, OpFoldResult> getDimAndTileMapping();
 
-    /// Return the tile sizes as OpFoldResult.
+    /// Return the tile sizes as OpFoldResult. Will return the Value
+    /// of the constant Op, not the constant Attribute.
```

Comment:
It's about time to fix it - could you add a TODO? Thanks!

Reply:
Not sure I follow what the changed comment is meant to imply.

Reply:
That in the following case, this method returns a `Value` corresponding to `%size` rather than a DenseAttribute with value `1`: 
```mlir
%size = arith.constant 1 : i32
```

It's probably worth clarifying further.

Reply:
Added this as an example